yarn.lock

```diff
@@ -3225,7 +3347,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@emotion/styled@npm:^10.0.23, @emotion/styled@npm:^10.0.27":
+"@emotion/styled@npm:^10.0.23":
```

Comment:
Looks like we will be adding `@emotion/styled@11.3.0` but `@emotion/styled@10.0.27` is still in the system. This applies to other `@emotion/*` packages in this PR too. 

This is not a bad thing _per se_, but it might indicate we still have Emotion v10 around. It might be worth double checking.

Reply:
I'll double check, but I think that comes from wordpress. The wp packages still require emotion 10. After our react 17 + wp update lands, those should also support emotion 11

Reply:
Yep, v10 comes from `@storybook/theming` and `@wordpress/components`, which are both expected